app/jobs/send_job_application_data_expiry_notification_job.rb

```diff
@@ -0,0 +1,19 @@
+class SendJobApplicationDataExpiryNotificationJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    start_of_today = DateTime.now.in_time_zone(Time.zone).beginning_of_day
+    end_of_today = DateTime.now.in_time_zone(Time.zone).end_of_day
+    two_weeks_less_than_a_year_ago = 1.year - 2.weeks
+
+    two_weeks_until_expiry = (start_of_today - two_weeks_less_than_a_year_ago)..(end_of_today - two_weeks_less_than_a_year_ago)
+
+    vacancies_with_two_weeks_until_expiry = Vacancy.expired.includes(organisations: :publishers).where(expires_at: two_weeks_until_expiry)
```

Comment:
for clear naming, can you do something like `vacancies_less_than_two_weeks_from_expiry`